### PR TITLE
feat: Add context for EH logic

### PR DESCRIPTION
**Pull request by BjarkeMeier**
# Description

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [ ] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

File: source/electrical_heating/electrical_heating/domain/calculation.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+)
+
+
+def execute(spark: SparkSession, args: ElectricalHeatingArgs) -> None:
+    # Create repositories to obtain data frames
+    electricity_market_repository = em.Repository(spark, args.catalog_name)
+    measurements_gold_repository = mg.Repository(spark, args.catalog_name)
+
+    # Read data frames
+    consumption_periods_df = (
```
Comment (JesperGomes) [inline source/electrical_heating/electrical_heating/domain/calculation.py:18]: didn't we agree to omit _df postfix?

Comment (BjarkeMeier) [reply source/electrical_heating/electrical_heating/domain/calculation.py]: You're right. Fixed.

**Review by JesperGomes — APPROVED** [review]
LGTM

File: source/electrical_heating/electrical_heating/domain/calculation.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        )
+        .agg(F.sum("quantity").alias("quantity"))
+        .select(
+            F.col("child.metering_point_id"),
+            F.col("date"),
+            F.col("quantity"),
+        )
+    )
+
+    return df
```
[the remark this answers is not in the document]
Comment (BjarkeMeier) [reply source/electrical_heating/electrical_heating/domain/calculation.py:79]: Definitely. If not sooner we'll fix it at the latest when we activate the first scenario test.
Other changes need to be made as well to pass that test.

**Pull request merged**